evaluate_library_access_from_output_tsv/stratefied_sample_of_dois_and_manual_full_text_check_results.tsv

```diff
@@ -0,0 +1,201 @@
+"doi"	"oadoi_color"	"full_text_indicator_automated"	"date_of_manual_full_text_check"	"full_text_indicator_manual"
```

Comment:
Also do you plan to do an in-network / out-of-network access check? If so, should column name (full_text_indicator_manual) reflect network status?

Reply:
Fleshing out my [comment](https://github.com/greenelab/library-access/issues/15#issuecomment-349685046) from #15, I wasn't planning to do the separate out-of-network check, but then I re-read [your comment](https://github.com/greenelab/library-access/issues/15#issuecomment-350377617), and see your point. For clarity (and mine/others' review), you noted:

> If all articles are freely available off campus, then Penn's subscriptions aren't providing crucial access.

However, this seems like a separate -- though still useful -- research question to our original one, which is re: the coverage of Penn's subscriptions (and, now, as a subset of that, the accuracy of our estimation of that coverage). If I'm reading this accurately, taking that underlying question to its full extent, it would be more about the coverage of someone going anywhere *except* Penn and SciHub (like with bronze-ish articles that are illegally/quasi-legally posted on their authors' home pages).

All this is to say that evaluating whether the Library's subscriptions are worthwhile *is a full-time job,* and there's someone doing exactly that here in our Libraries. Thus, if we do a separate out-of-network access check:

- If it's for checking the accuracy of our automated means, I don't think it will yield additional information over the on-campus manual check -- though I recognize that I may not be conceptualizing it as you are, and would like to be on the same page.
- If it's for this additional question, of whether Penn's subscriptions are "providing crucial access," I think that we should think more carefully about that, as it's a big question that would require separate analyses from what we're dealing with in this PR (and in the scope of what we've been discussing in this whole branch of the manuscript project)[1], and which would likely benefit from additional input.

Does that make sense, as I'm explaining it here? If so, would you be open to tabling the idea of the out-of-network check, and sticking with the in-network check?

[1] For example, we might want a larger sample of DOIs if we're answering that question, and would need to separately estimate the rate of Library superfluous access.

Reply:
Having both an in-network and out-of-network availability call will result in the following categories:

1. Available in network, Not available out of network
2. Available in network, available out of network
3. Not available in network, not available out of network
4. Not available in network, available out of network. Let's assume this will never happen

1 is where Penn's subscription is critical for access. If we encounter `full_text_indicator=false` for any category 1 articles then that is a concerning error.

2 will occur when the article is freely available from the publisher. I'm assuming that sometimes we'll see `full_text_indicator=false` for these category 2 DOIs, due to reasons like:

+ the journal is OA but not part of the database that enables calling `full_text_indicator=true`
+ the journal is subscription, Penn does not subscribe, but the article is not paywalled

3 represents paywalled articles where Penn doesn't have access. If we encounter `full_text_indicator=true` here that is a concerning error.

So I think we'll be able to assess the meaning and accuracy of `full_text_indicator` much better if we can differentiate categories 1-3 above. Make sense?

Reply:
> Can you just select 100 DOIs from each without using oaDOI calls?

Yes -- I've implemented this in 6af5b03. My understanding following your [later comment](https://github.com/greenelab/library-access/issues/15#issuecomment-350378365) had been that you would be ok with the sub-stratification within `full_text_indicator`.

Reply:
Yes, it does; thank you. Implemented in 8ec97920.